docker/README.md

```diff
@@ -212,7 +212,7 @@ docker ps
 2. Bind and get the bash prompt of running container
 
 ```bash
-docker exec -it <container_name> /bin/bash
+docker exec -it -u="root" <container_name> /bin/bash
```

Comment:
@AshwinRaikar88 Can you please explain why the root user needs to be set here?

Reply:
@AshwinRaikar88 both volumes are mounted with model-server as owner. So `-u root` is not needed.